src/main/kotlin/app/revanced/patches/idaustria/patch/RootDetectionPatch.kt

```diff
@@ -0,0 +1,22 @@
+package app.revanced.patches.idaustria.patch
+
+import app.revanced.patcher.annotation.*
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.idaustria.fingerprints.RootDetectionFingerprint
+
+@Patch
+@Name("root-patch")
```

Comment:
Apply this change to the class name as well.
```suggestion
@Name("remove-root-detection")
```